maven-setup/action.yml

```diff
@@ -56,6 +56,20 @@ inputs:
   develocity-allow-untrusted-server:
     description: 'Whether to allow communicating with untrusted server'
     required: false
+  develocity-enforce-url:
+    description: 'Whether to enforce the URL over any defined locally to the project'
+    required: false
+  develocity-capture-file-fingerprints:
+    description: 'Whether to enable/disable capturing of file fingerprints. This setting is enabled by default.'
```

Comment:
Is this really required?
The maven-setup action is not compatible with the GE extension and requires DV for the capture scenario (involving the Maven extension).
Could we expand that to the injection as well?
If we only supports DV, then the capture is enabled by default

Reply:
True, although we allow configuring this in other CI Plugins. Would you say that having an ability to turn this off is redundant?

Reply:
nvm, let's keep it for the sake of alignment.